packages/hooks/src/createRunHook.js

```diff
@@ -30,6 +30,14 @@ function createRunHook( hooks, returnFirstArg ) {
 
 		const handlers = hooks[ hookName ].handlers;
 
+		// The following code can be stripped from production builds.
+		/* develblock:start */
+		// Handle any 'all' hooks registered.
+		if ( hooks.all ) {
+			handlers.push( hooks.all.handlers );
```

Comment:
You're pushing an array into the array, rather than the members of that array.

i.e.

```
handlers = [ 1 ];
hooks.all.handlers = [ 2 ];

handlers.push( hooks.all.handlers );

console.log( handlers );
// [ 1, [ 2 ] ]
```

You'd want one of either:

```js
handlers = handlers.concat( hooks.all.handlers );
handlers.push( ...hooks.all.handlers );
```

Reply:
eagle eyes! i wouldn't have caught that error until i wrote the tests. will fix

Reply:
this is fixed.